package.json

```diff
@@ -56,21 +57,25 @@
     "parcel-plugin-bundle-visualiser": "^1.2.0",
     "rimraf": "^3.0.2",
     "send-ranges": "^3.0.0",
-    "serve-static": "^1.14.1"
+    "serve-static": "^1.14.1",
+    "typescript": "^4.3.5"
   },
   "scripts": {
     "build": "npm run build:clean; run-p build:browser build:node;",
     "build:clean": "rimraf dist-node/ dist-browser/",
-    "build:node": "parcel build src/geotiff.js --target node --out-dir dist-node/",
-    "build:browser": "parcel build src/geotiff.js --target browser --out-dir dist-browser/ --global GeoTIFF --public-url .",
+    "build:node": "parcel build src/geotiff.js --target node --out-dir dist-node/ && npm run build:node:types",
+    "build:node:types": "cp src/geotiff.d.ts dist-node/geotiff.d.ts",
```

Comment:
So you don't need these build scripts (`build:node:types` & `build:browser:types`). Typescript only respects one `types` declaration.

Reply:
dropped